### DISCUSSION TITLE
TradeskillMaster ChromieCraft (RP)

### DISCUSSION THREAD

**Discussion by Jmths** [chromiecraft/chromiecraft#2923] [General]
How do I get the TradeSkillMaster to work? I'd like this specific addon for Auction House related practise to work.
I was hoping a smart fellow around here could help me out, picture related: https://i.imgur.com/M0484h7.png

Best regards

j.m.thomas@hotmail.nl
Jesse / Chouette
![TradeSkillMaster ChromieCraft](https://user-images.githubusercontent.com/67026712/152140558-3b63efb8-0705-4172-9ee5-1691e122acb2.png)
e

**Comment by Annamaria-CC** [comment]
Hey, I am guessing you are using a wrong version of the addon. Try out https://github.com/Irame/TSM_WOTLK/archive/master.zip this one
